### PR TITLE
Add the __init__.py files

### PR DESCRIPTION
**Pull request by chawki27000**
I added in each python source code folder __init__.py file in the purpose to create a python package

### CODE REVIEW

**Comment by iblancasa** [comment]
Hi @chawki27000,

Could you follow the described structure in this links?
https://stackoverflow.com/questions/1944569/how-do-i-write-good-correct-package-init-py-files
http://mikegrouchy.com/blog/2012/05/be-pythonic-__init__py.html

Thank you :+1:

**Comment by chawki27000** [comment]
Hi @iblancasa 
I will do that.

File: src/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+__all__ = ['manualtest', 'githubcity']
```
Comment (iblancasa) [inline src/__init__.py]: Could you remove this file? This is not a Python package.

File: src/githubcity/__init__.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+from ghcity import GitHubCity
```
Comment (iblancasa) [inline src/githubcity/__init__.py]: Could you disable the linter warnings?

Comment (cxong) [reply src/githubcity/__init__.py]: `# pylint: disable=unused-import`

Comment (iblancasa) [reply src/githubcity/__init__.py]: Yes

**Comment by iblancasa** [comment]
Hi @chawki27000, could you apply the requested changes?

**Pull request merged**